operations/evaluate.go

```diff
@@ -50,6 +57,12 @@ func Evaluate() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "loading project")
 			}
+			if diffable {
+				sortTasksByName := model.ProjectTasksByName(p.Tasks)
+				sort.Sort(sortTasksByName)
+				p.Tasks = sortTasksByName
+				sort.Sort(p.BuildVariants)
```

Comment:
This looks like it would sort by display name, but `ProjectTasksByName` sorts by task name. Can we make them consistent and sort by name (or display name) for both of them? There's a `model.BuildVariantsByName` if it helps.

Reply:
they are both by name now!